kobo/apps/superuser_stats/tasks.py

```diff
@@ -484,6 +495,27 @@ def generate_user_statistics_report(
         ).order_by('user__date_joined').annotate(count_sum=Sum('counter'))
     )
 
+    # get NLP statistics
+    nlp_counters = (
+        MonthlyNLPUsageCounter.objects.annotate(
+            date=Cast(
+                Concat(F('year'), Value('-'), F('month'), Value('-'), 1),
+                DateField(),
+            ),
+        ).filter(date__range=(start_date, end_date)).values(
+            'user_id',
+        ).annotate(
+            total_google_asr=Sum(
+                Cast(KeyTextTransform('google_asr_seconds',
+                                      'counters'), IntegerField()),
+            ),
+            total_google_mt=Sum(
+                Cast(KeyTextTransform('google_mt_characters',
+                                      'counters'), IntegerField()),
+            ),
```

Comment:
Even if it's a good solution to use `KeyTextTransform` (I did not know about it, thank you), I think, it could be simplified with `F()` 

```python 
            total_google_asr=Sum(
                Cast(F('counters__google_asr_seconds'), IntegerField()),
            ),
            total_google_mt=Sum(
                Cast(F('counters__google_mt_characters'), IntegerField()),
            ),
```